kafka/modules/kafka/3.4.0/module.yaml

```diff
@@ -8,7 +8,7 @@ envs:
     value: "amqstreams-kafka-34-container"
 
 artifacts:
-  - md5: 11c4eb691ae1965b39b7137b894d75ab
+  - md5: 15dcea9f299b4ba03256e85b222acf31
```

Comment:
this is wrong you got the MD5 for the pom file not the zip. It should be `7891c4a71a2727906f60cf1cf071c4b0`

Reply:
Fixed